packages/terra-date-input/tests/wdio/date-input-spec.js

```diff
@@ -75,13 +75,12 @@ Terra.describeViewports('Date Input', ['medium'], () => {
   });
 
   describe('Month Select placeholder option clears value', () => {
-    it('Selects February in month select', () => {
+    it('Selects January in month select', () => {
```

Comment:
This issue is going to reoccur when the current month is January again. We should update the tests so that they are not affected by the current date.

Reply:
This issue will be fixed as part of this PR https://github.com/cerner/terra-framework/pull/2019#discussion_r1482642514